air/src/air/coefficients.rs

```diff
@@ -65,7 +65,9 @@ impl<E: FieldElement> Default for AuxTraceRandElements<E> {
 #[derive(Debug, Clone)]
 pub struct ConstraintCompositionCoefficients<E: FieldElement> {
     pub transition: Vec<E>,
+    pub lagrange_kernel_transition: Vec<E>,
     pub boundary: Vec<E>,
+    pub lagrange_kernel_boundary: Option<E>,
 }
```

Comment:
What benefits do you see in doing this? I typically don't like this style, since the type becomes much less descriptive, and more bug-prone. To understand what the `lagrange` field is, I have to go see how it is populated in the code. And I need to keep in my head that `lagrange[0]` is different from all other `lagrange[i]`, and lookout for bugs e.g. that loop over all indices and apply a "transition constraint operation".

Reply:
Replaced both fields with a single `LagrangeConstraintsCompositionCoefficients` one